[OSDOCS#9221]: 4.14.8 errata/rn (#69910)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

[id="ocp-4-14-8"]
=== RHSA-2024:0050 - {product-title} 4.14.8 bug fix and security update

Issued: 2024-01-09

> {product-title} release 4.14.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0050[RHBA-2024:0049] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0052[RHBA-2024:0052] advisory.

Reviewer: The link is 0050 but is showing as 0049. I'm assuming 0050 is correct here?

```suggestion
{product-title} release 4.14.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0050[RHBA-2024:0050] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0052[RHBA-2024:0052] advisory.
```

* Previously, customers could not leverage OpenShift's Azure Identity capability for short-lived authentication tokens with layered products. With this release, OLM-managed operators have increased security by enabling this support. (link:https://issues.redhat.com/browse/OCPBUGS-25275[*OCPBUGS-25275*])

[id="ocp-4-14-8-bug-fixes"]
==== Bug fixes

* Previously, installation would fail when installing {product-title} with the `bootMode` set to `UEFISecureBoot` on a node where Secure Boot is disabled. With this release, subsequent attempts to install {product-title} with Secure Boot enabled will proceed normally.(link:https://issues.redhat.com/browse/OCPBUGS-19884[*OCPBUGS-19884*])

> * Previously, the installer would fail to destroy a cluster when using regional PDs in google cloud.

Reviewer: I'm not sure if this should be "Google Cloud Platform", but at the very least should be capitalized as Google Cloud.

```suggestion
* Previously, the installer would fail to destroy a cluster when using regional PDs in Google Cloud.
```

> With this release, the replicated zones were found and the disks were deleted. (link:https://issues.redhat.com/browse/OCPBUGS-22770[*OCPBUGS-22770*])

Reviewer: The tense here seems off to me.

```suggestion
With this release, the replicated zones are found and the disks are deleted properly. (link:https://issues.redhat.com/browse/OCPBUGS-22770[*OCPBUGS-22770*])
```

> * Previously, `additionalSecurityGroupIDs` were not being utilized by the control plane nodes, and were only specified in the `defaultMachinePlatform` stanza. With this release, the value from the `defaultMachinePlatform` is used when there is no specification for the control plane nodes. (link:https://issues.redhat.com/browse/OCPBUGS-22771[*OCPBUGS-22771*])

Reviewer: I don't think that this rewrite holds the same meaning as the original text in the jira.

How I understand it is that if the additionalSecurityGroupIDs was specified in the defaultMachinePlatform stanza (but not in the control plane nodes), it wasn't used at all. But with this fix, now it is.
